vc-generator/secret.js

```diff
@@ -1,33 +1,19 @@
 /*!
  * Copyright 2023-24 Digital Bazaar, Inc. All Rights Reserved
  */
-import {createRequire} from 'node:module';
-import {cryptosuites} from './constants.js';
-
-const require = createRequire(import.meta.url);
+import * as Ed25519Multikey from '@digitalbazaar/ed25519-multikey';
 
 // This is only used inside this test suite for generating vcs for the verify
 // proof tests.
-export const TEST_KEY_SEED =
-  'z1AZVaiqEq3kXaf4DJD5qXUfdJBFbW1JNe4FF58HwMgVE6u';
-
-export async function getKeyPair({suiteName, keyType}) {
-  const {serializedKeys, multikey} = cryptosuites.get(suiteName);
-  const keyDir = './keys';
-  if(!serializedKeys) {
-    throw new Error(`Unrecognized suite: ${suiteName}`);
-  }
-  if(typeof serializedKeys === 'string') {
-    return multikey.from(require(`${keyDir}/${serializedKeys}`));
-  }
-  const keyPath = serializedKeys[keyType];
-  if(!keyPath) {
-    throw new Error(`Unrecognized keyType ${keyType} for suite ${suiteName}`);
-  }
-  return multikey.from(require(`${keyDir}/${keyPath}`));
-}
+const TEST_KEY_PAIR = {
+  id: 'did:key:z6MkwXG2WjeQnNxSoynSGYU8V9j3QzP3JSqhdmkHc6SaVWoT#z6MkwXG2Wje' +
+    'QnNxSoynSGYU8V9j3QzP3JSqhdmkHc6SaVWoT',
+  '@context': 'https://w3id.org/security/multikey/v1',
+  type: 'Multikey',
+  controller: 'did:key:z6MkwXG2WjeQnNxSoynSGYU8V9j3QzP3JSqhdmkHc6SaVWoT',
+  publicKeyMultibase: 'z6MkwXG2WjeQnNxSoynSGYU8V9j3QzP3JSqhdmkHc6SaVWoT',
+  secretKeyMultibase: 'zrv3rbPamVDGvrm7LkYPLWYJ35P9audujKKsWn3x29EUiGwwhdZ' +
```

Comment:
It'd be great to always leave a comment anywhere we store "secret" key material in public. Just something that explains that this is for testing only, don't use it, etc. Robots (and humans) will trip over this at some point, and when they do, we should be ready to explain why this is an exception to the "don't keep secrets in git" rule.

Reply:
I improved the comment here: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/pull/39/commits/09184ae4269bb6f113e3c94528bba8e6e8663f50